pyproject.toml

```diff
@@ -19,7 +19,7 @@ classifiers = [
 
 [tool.poetry.dependencies]
 python = "^3.7"
-qiskit = "^0.27.0"
+qiskit = "^0.34.0"
```

Comment:
Is bumping the minimum qiskit required? Maybe we can do `>=0.27, <0.35` for flexibility.

Reply:
Yeah happy to do that, I just updated it similar to whats on the pre-release branches